frame/support/procedural/src/lib.rs

```diff
@@ -69,6 +69,12 @@ fn get_cargo_env_var<T: FromStr>(version_env: &str) -> std::result::Result<T, ()
 	T::from_str(&version).map_err(drop)
 }
 
+/// Generate the counter_prefix related to the storage.
+/// counter_prefix is used by counted storage map.
+fn counter_prefix(prefix: &str) -> String {
```

Comment:
@bkchr couldn't find a better place tbh, felt wrong to create it's own file for it, seeing as how `lib.rs` already has a couple of helpers. If we keep moving stuff to be more general - might be worth actually considering it's own file.